python/cudf/cudf/_lib/copying.pyx

```diff
@@ -598,45 +467,33 @@ def boolean_mask_scatter(list input_, list target_columns,
         return []
 
     if isinstance(input_[0], Column):
-        return _boolean_mask_scatter_columns(
-            input_,
-            target_columns,
-            boolean_mask
-        )
+        plc_input = pylibcudf.Table([col.to_pylibcudf(mode="read") for col in input_])
```

Comment:
It's not reserved per se. It's a built-in function as opposed to a keyword like `in`, so you can shadow it. It's only really an issue if you actually needed that function, but I don't know if it might confuse some linters (I haven't observed that in a long time though) and it can lead to odd syntax highlighting in editors. I can change it if you'd like, no strong preference.

Reply:
Also no longer relevant as above.